cli/cmd/stat.go

```diff
@@ -53,16 +53,19 @@ func newCmdStat() *cobra.Command {
   Examples:
   * deploy
   * deploy/my-deploy
-  * deploy my-deploy
-  * ns/my-ns
+  * rc/my-replication-controller
+	* ns/my-ns
+	* authority
+	* au/my-authority
   * all
 
 Valid resource types include:
 
   * deployments
   * namespaces
   * pods
-  * replicationcontrollers
+	* replicationcontrollers
+	* authority (not supported in --from)
   * services (only supported if a "--from" is also specified, or as a "--to")
```

Comment:
Can you remove the quotes around `--from` and `--to` on this line, for consistency with how those flags are referenced for other resources in this list?

Reply:
ahhh whooops sorry missed this from before!